salt/modules/zypperpkg.py

```diff
@@ -1717,9 +1704,7 @@ def upgrade(
     dryrun=False,
     dist_upgrade=False,
     fromrepo=None,
-    novendorchange=False,
```

Comment:
same here. you cannot remove kwargs since someone can be calling this function with positional args.